### DISCUSSION TITLE
Unit8list support

### DISCUSSION THREAD

**Discussion by dbaanils** [deepgram/community#195] [General help] [answered]
We are currently integrating the Deepgram transcript API into our mobile application. However, we are facing a challenge where we can only send data in the form of a Uint8List from our mobile application, preventing us from establishing a WebSocket connection with the Deepgram API.

We would greatly appreciate your assistance in finding a solution to this issue. Is it possible to modify the API to accept Uint8List data as well?

**Comment by jjmaldonis** [comment]
Hey @dbaanils, can you explain a bit more about how you are making the WebSocket connection to Deepgram?

We are guessing you are building the mobile app in flutter since you mentioned a Uint8List. Have you seen the flutter tutorial we have? https://blog.deepgram.com/flutter-speech-to-text-tutorial/  The tutorial is slightly out of date (we are working on updating it), but it has useful information about how to stream audio to Deepgram.

If the flutter tutorial isn't helpful, can you include the code you're using to connect to Deepgram so we can understand how you're using the Uint8List?

**Reply by jjmaldonis** [reply, marked as answer]
For anyone else reading this post, the flutter tutorial has been updated.
